app/components/search/SearchEntry.jsx

```diff
@@ -7,6 +7,21 @@ import { RelativeOpeningTime } from '../listing/RelativeOpeningTime';
 import './SearchEntry.scss';
 
 class SearchEntry extends Component {
+  // eslint-disable-next-line consistent-return
```

Comment:
We probably shouldn't disable this. If we hit this case, then we'll silently not return anything. If we fall out of all of the `if` checks, then we probably want to return some "No address found" message.